### PR TITLE
Services

### PR DESCRIPTION
**Pull request by atsanda**
Review my first steps, please.

### CODE REVIEW

File: src/main/java/ru/javafiddle/web/services/UserService.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import javax.ws.rs.core.UriInfo;
+
+/**
+ * Created by artyom on 15.11.15.
+ */
+@Path("/users")
+public class UsersService {
+
+    @EJB
+    private UserBean userBean;
```
Comment (rundb) [inline src/main/java/ru/javafiddle/web/services/UserService.java:21]: If you created this bean, it should be committed either.

File: src/main/java/ru/javafiddle/web/services/UserService.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+    @POST
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response register(User user, @Context UriInfo uriInfo) throws UriBuilderException {
+        User newUser = userBean.register(user));
```
Comment (rundb) [inline src/main/java/ru/javafiddle/web/services/UserService.java:27]: Does registration include validation?

**Comment by rundb** [comment]
LGTM

**Comment by atsanda** [comment]
I thpought that validation must be on the client side.

**Comment by rundb** [comment]
Okay, what do you mean by "client validation"?
I supposed steps like e-mail verification, checking if e-mail or username is already registered - these things cannot be checked on client side

**Comment by atsanda** [comment]
I meant some obvious verifications, such as email without "@". Of course,
checking for uniqueness must be on the server side and I will take it into
consideration.

2015-11-16 8:46 GMT+03:00 Roman Turkin notifications@github.com:

**Pull request merged**